Cloud Application Manager/Datacenter Edition/camdce-with-ec2-instance-role.md

```diff
@@ -104,7 +104,7 @@ In this example the policy is named *CAMApplianceAssumeRolePolicy*.
 
 ![AWS Console Role review](../../images/appliance/aws-assume-role-policy-role-review.png)
 
-### Attach the IAM Role to the Cloud Application Manager Data Center Edition EC2 Instance
+### Attach the IAM Role to the Cloud Application Manager Datacenter Edition EC2 Instance
```

Comment:
Please, revert this change and leave it as two words in all places

Reply:
done